<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostsDialog</class>
 <widget class="QDialog" name="HostsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>379</width>
     <height>397</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>List of Available Hosts</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listHosts"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnRemoveHost">
      <property name="text">
       <string>Remove host</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Add host name or ip address and press &quot;Add host&quot; to add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editNewHost">
      <property name="text">
       <string extracomment="Enter ip address (e.g. 192.168.1.54) or host Name"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAddHost">
      <property name="text">
       <string>Add host</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
